### PR TITLE
fix: fix wrong import module path

### PR DESCRIPTION
**Pull request by ColeDrain**
- fix wrong import for in main.go
- change import path for go-zopfli

### CODE REVIEW

**Comment by ColeDrain** [comment]
I see one of the goals of this project is: "We aim for simplicity with the least external dependencies"

Do we plan on adding the go-zopfli library directly to the project ?

File: core/core.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	"time"
 	"unsafe"
 
-	"github.com/kislerdm/chatgpt-c4/core/go-zopfli/zopfli"
+	"github.com/foobaz/go-zopfli/zopfli"
```
Comment (kislerdm) [inline core/core.go:12]: **(issue)** Please change it to `github.com/kislerdm/diagramastext/core/go-zopfli/zopfli`, it is a git submodule relying on a fork of `github.com/foobaz/go-zopfli/zopfli`. Thanks!

```suggestion
"github.com/kislerdm/diagramastext/core/go-zopfli/zopfli"
```
Please clone the repo using the command:
```commandline
git clone --recurse-submodules git@github.com:kislerdm/diagramastext.git
```

**(todo)** add the clone instruction to readme

**Review by kislerdm — CHANGES_REQUESTED** [review]
Hey Ugo! Welcome aboard and congrats with your first contribution! Thanks a lot for your support! :) Please consider the comments before PR can be merged.

**Comment by kislerdm** [comment]
> Do we plan on adding the go-zopfli library directly to the project ?

I'd like to suggest to follow the status quo: please consider the [comment](https://github.com/kislerdm/diagramastext/pull/16#discussion_r1096693449) for the context. We can re-evaluate the decision at a later point. Thanks!

**Pull request closed without merging**